### DISCUSSION TITLE
cnn을 이용한 mnist 분류 코드 레이어 질문

### DISCUSSION THREAD

**Discussion by keemdy** [Boostcamp-AI-Tech-1-15/ddobokki_log#22] [2. DL Basic]
```
def forward(self, x):
        x = self.conv1(x)
        x = F.relu(x)
        x = self.conv2(x)
        x = F.relu(x)
        x = F.max_pool2d(x, 2)
        x = self.dropout1(x)
        x = torch.flatten(x, 1)
        x = self.fc1(x)
        x = F.relu(x)
        x = self.dropout2(x)
        x = self.fc2(x)
        output = F.log_softmax(x, dim=1)
        return output
```
[https://github.com/pytorch/examples/blob/master/mnist/main.py](url)

참고한 코드에서는 첫번째 convolutional layer이후에는 pooling layer가 없는데 그 이유가 뭘까요 ? fc layer 전에 한 번만 있으면 되는 걸까요 ?
parameter 수를 줄이기 위해서는 subsampling이 필요한데 왜 넣지 않았을까요 ?

**Comment by mybirth0407** [comment]
MNIST에서는 저 정도 모델로도 학습이 잘 되기 때문에 그런 듯 합니다.. 모델의 구조에 대해서는 대개 만든 사람의 의도가 절대적이기 때문에.. 이해하기 어려운 경우가 많고, 위의 경우는 그냥 간단하게 모델을 구현하고자 했던 듯.. 하네요..!
